velox/docs/functions/spark/string.rst

```diff
@@ -89,6 +89,16 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
 
     Returns the length of ``string`` in characters.
 
+.. spark:function:: levenshtein(string, string[, threshold]) -> integer
+
+    Returns the `Levenshtein distance <https://en.wikipedia.org/wiki/Levenshtein_distance>`_ between the two given strings. If
+    ``threshold`` is set and the levenshtein distance exceeds it, return -1.
+    Negative threshold is allowed, return -1. ::
```

Comment:
Document suggestion: If the provided \``threshold\`` is negative, or the levenshtein distance exceeds \``threshold\``, returns -1.